From 5d01aa2b7f1ccd02f91acc117fd53622154adc36 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Thu, 30 Oct 2008 23:23:16 +1100
Subject: [PATCH] qemu: seems the move to the svn rev of qemu broke as there are compiler bugs

svn qemu has compiler bugs and wont build (on ubuntu hardy), so rever back to
the old 0.9.1 that works.
---
 conf/distro/include/sane-srcrevs.inc |    8 ++++----
 packages/qemu/qemu-native_svn.bb     |    2 ++
 packages/qemu/qemu_svn.bb            |    2 ++
 3 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/conf/distro/include/sane-srcrevs.inc b/conf/distro/include/sane-srcrevs.inc
index 0fbf62b..ae914c8 100644
--- a/conf/distro/include/sane-srcrevs.inc
+++ b/conf/distro/include/sane-srcrevs.inc
@@ -180,10 +180,10 @@ SRCREV_pn-python-gsmd = "148"
 SRCREV_pn-python-lightmediascanner = "68"
 SRCREV_pn-python-pyrtc = "d1f32060474767582dd4248d5e5c2adc63931d5a"
 SRCREV_pn-python-pyyaml = "294"
-QEMUSRCREV = "4242"
-SRCREV_pn-qemu-native ?= "${QEMUSRCREV}"
-SRCREV_pn-qemu-sdk ?= "${QEMUSRCREV}"
-SRCREV_pn-qemu ?= "${QEMUSRCREV}"
+#QEMUSRCREV = "4242"
+#SRCREV_pn-qemu-native ?= "${QEMUSRCREV}"
+#SRCREV_pn-qemu-sdk ?= "${QEMUSRCREV}"
+#SRCREV_pn-qemu ?= "${QEMUSRCREV}"
 SRCREV_pn-remoko = "115"
 SRCREV_pn-remoko-server = "115"
 SRCREV_pn-s3c24xx-gpio ?= "4130"
diff --git a/packages/qemu/qemu-native_svn.bb b/packages/qemu/qemu-native_svn.bb
index 1e3a4db..60f5a39 100644
--- a/packages/qemu/qemu-native_svn.bb
+++ b/packages/qemu/qemu-native_svn.bb
@@ -1,2 +1,4 @@
 require qemu_svn.bb
 require qemu-native.inc
+
+DEFAULT_PREFERENCE = "-99"
diff --git a/packages/qemu/qemu_svn.bb b/packages/qemu/qemu_svn.bb
index 6e52a15..44d07a6 100644
--- a/packages/qemu/qemu_svn.bb
+++ b/packages/qemu/qemu_svn.bb
@@ -3,6 +3,8 @@ DEPENDS = "zlib"
 PV = "0.9.1+svnr${SRCREV}"
 PR = "r12"
 
+DEFAULT_PREFERENCE = "-99"
+
 FILESPATH = "${FILE_DIRNAME}/qemu-${PV}/:${FILE_DIRNAME}/qemu-0.9.1+svn/"
 
 SRC_URI = "\
-- 
1.5.4.3

